### PR TITLE
APP-4546 Button bug fixes

### PR DESCRIPTION
**Pull request by symphony-annaLlorens**
### Fixes:
- Padding on small and large buttons
- Destructive well displayed on Styles
- Tertiary Accent instead of Active
- Color palette on dark mode
- Loading large button padding was too small

### Demo

Screenshot 2021-09-28 at 11 50 04

Screenshot 2021-09-28 at 11 49 55

### CODE REVIEW

File: src/components/atoms/_button.scss [hunk 1]
```diff
@@ -19,6 +19,9 @@ $tk-btn-size: toRem(32);
   i {
     font-size: toRem(16);
   }
+  &.loading {
+    min-width: toRem(79);
```
Comment (AliciaSymphony) [inline src/components/atoms/_button.scss:23]: What is the use of a min width on loading?

File: src/utils/mixins/_buttons.scss [hunk 2]
```diff
@@ -60,7 +60,6 @@
     color: map-get($colorMap, textDisabled);
   }
   &.loading {
-    min-width: toRem(79);
```
Comment (symphony-adnane) [inline src/utils/mixins/_buttons.scss:63]: why moving it out from here?

**Review by symphony-adnane — APPROVED** [review]

File: src/utils/mixins/_buttons.scss (hunk 2 above)
Comment (symphony-annaLlorens) [reply src/utils/mixins/_buttons.scss:63]: because it was repeated on 2 different mixins

File: src/components/atoms/_button.scss (hunk 1 above)
Comment (symphony-annaLlorens) [reply src/components/atoms/_button.scss:23]: For the icon buttons

Comment (symphony-annaLlorens) [reply src/components/atoms/_button.scss]: no used anymore

File: src/variables/themes/_dark.scss [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
+  --tk-button-color-primary-disabled: #{getColor($electricity, '70')};
+  --tk-button-color-primary-text-disabled: #{getColor($electricity, '60')};
+  --tk-button-color-destructive-disabled: #{getColor($red, '70')};
+  --tk-button-color-destructive-text-disabled: #{getColor($red, '60')};
+
   // Secondary btn
-  --tk-btn-color-tertiary-default: #{getColor($graphite, '30')};
-  --tk-btn-color-secondary-hover: transparent;
-  --tk-btn-color-secondary-active: transparent;
+  --tk-btn-color-secondary-default: #{getColor($graphite, '30')};
```
Comment (symphony-jean-michael) [inline src/variables/themes/_dark.scss:10]: Why do we use 'button' in the Primary button and 'btn' in the Secondary button?
Example:  '--tk-button-color...' and  '--tk-btn-color' 
Can we use always the same name '--tk-button-...'?

**Review by symphony-jean-michael — APPROVED** [review]

**Pull request merged**